qmat/solvers/dahlquist.py

```diff
@@ -0,0 +1,606 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+r"""
+Solvers for the Dahlquist equation based on :math:`Q` coefficients,
+also implementing SDC sweeps with given :math:`Q_\Delta` coefficients.
+"""
+import numpy as np
+
+
+class Dahlquist():
```

Comment:
Why is there a special Dahlquist solver? Why not use generic SDC solver with Dahlquist "diffop"?